include/RDM/BaseRDMBuilder.hpp

```diff
@@ -0,0 +1,52 @@
+#ifndef GQCG_BASERDMBUILDER_HPP
+#define GQCG_BASERDMBUILDER_HPP
+
+
+#include "RDM/OneRDM.hpp"
+#include "RDM/TwoRDM.hpp"
+#include "RDM/RDMs.hpp"
+#include "FockSpace/BaseFockSpace.hpp"
+
+
+namespace GQCG {
+
+
+/**
+ *  BaseRDMBuilder is an abstract base class for the calculation of a density matrix from a given wave function
+ *  or coefficient expansion in a Fock space
```

Comment:
Adding the relationship between `BaseRDMBuilder` and `RDMBuilder` in this class documentation would be beneficial.

Maybe we could rename `RDMBuilder` because the name feels like `RDMBuilder` is a derived class of `BaseRDMBuilder`.

Reply:
Agreed